mmv1/third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -802,14 +803,23 @@ func ResourceComposerEnvironment() *schema.Resource {
 							},
 						},
 						"environment_size": {
-              Type:         schema.TypeString,
-              Optional:     true,
-							Computed:			true,
-              ForceNew:     false,
-              AtLeastOneOf: composerConfigKeys,
-              ValidateFunc: validation.StringInSlice([]string{"ENVIRONMENT_SIZE_SMALL", "ENVIRONMENT_SIZE_MEDIUM", "ENVIRONMENT_SIZE_LARGE"}, false),
-              Description:  `The size of the Cloud Composer environment. This field is supported for Cloud Composer environments in versions composer-2.*.*-airflow-*.*.* and newer.`,
-            },
+							Type:         schema.TypeString,
+							Optional:     true,
+							Computed:     true,
+							ForceNew:     false,
+							AtLeastOneOf: composerConfigKeys,
+							ValidateFunc: validation.StringInSlice([]string{"ENVIRONMENT_SIZE_SMALL", "ENVIRONMENT_SIZE_MEDIUM", "ENVIRONMENT_SIZE_LARGE"}, false),
+							Description:  `The size of the Cloud Composer environment. This field is supported for Cloud Composer environments in versions composer-2.*.*-airflow-*.*.* and newer.`,
+						},
+						"resilience_mode": {
+							Type:         	schema.TypeString,
+							Optional:		true,
+							Computed:		true,
+							ForceNew:		true,
+							AtLeastOneOf: composerConfigKeys,
+							ValidateFunc: validation.StringInSlice([]string{"RESILIENCE_MODE_UNSPECIFIED", "HIGH_RESILIENCE"}, false),
```

Comment:
We typically don't need `*_UNSPECIFIED` values in Terraform, because the user can just not include the field in the config. Do you know if it is necessary in this case?

Reply:
I removed the `*_UNSPECIFIED` value